eng/tools/TypeSpecValidation/src/TypeSpecValidation.ts

```diff
@@ -69,7 +69,7 @@ export async function main() {
   }
 
   // Format parent folder to include shared files
-  await runCmd(`npx tsp format ../**/*.tsp`, folder);
+  await runCmd(`npx tsp format ..${path.sep}**${path.sep}*.tsp`, folder);
```

Comment:
Is this change necessary?  I would expect `node` commands on Windows to work correctly with forward slashes.

Reply:
I would as well